docs/usage/self-hosting.md

```diff
@@ -200,7 +208,7 @@ Configure it either as `token` in your `config.js` file, or in environment varia
 
 #### GitLab CE/EE
 
-First, [create a personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account (select "api" scope).
+First, [create a personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account (select `read_user`, `api` and `write_repository` scopes).
```

Comment:
not sure, but i think `api` includes `write_repository` but not `read_user`

Reply:
No harm including all but I think you are right